packages/node/src/plugins/LoadFileChunkLoadingRuntimeModule.ts

```diff
@@ -253,12 +254,12 @@ class ReadFileChunkLoadingRuntimeModule extends RuntimeModule {
                       Template.indent([
                         loadScriptTemplate,
 
-                        `console.log('needs to load remote module from', ${JSON.stringify(
+                        this.options.verbose ? `console.log('needs to load remote module from', ${JSON.stringify(
```

Comment:
instead of changing all of the places of console.log, what do you think about creating a get logger function and in case of verbose, the function will be noop func, otherwise, console.log

Reply:
Sounds like a good idea!